app/forms/candidate_interface/other_qualification_details_form.rb

```diff
@@ -199,6 +199,8 @@ def qualification_needs_grade_sanitized?
     end
 
     def award_year_is_within_acceptable_range
+      return if award_year.to_i.zero? && award_year != '0'
```

Comment:
Non numeric characters, e.g `'abcde'. to_i` will transform to numeric 0, not within the acceptable year range. It was to avoid the validation being triggered in the test, giving two error messages. But now the use of the `in:` avoids having the custom validation, and the test only gives the correct error message. Something to do with how custom/in-built validations work in Rails 🤷‍♂️